packages/mux-uploader-react/src/index.tsx

```diff
@@ -11,11 +11,26 @@ import useObjectPropEffect from './useObjectPropEffect';
 export type MuxUploaderRefAttributes = MuxUploaderElement;
 
 export type MuxUploaderProps = {
+  endpoint?: string;
   type?: string;
   status?: boolean;
-  style?: CSSProperties;
+  style?: CSSProperties & {
+    ['--uploader-font-family']?: CSSProperties['fontFamily'];
```

Comment:
thought(non-blocker): This is fine for now, but we'll probably need to actually move these to a more "globally-scoped" definition, since you can apply the css vars at various levels of the CSSOM and they'll get inherited (common practice in e.g. media chrome). I completely spaced on that when we paired. I'll be sure to re-hunt down the declaration space needed for that.